src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,153 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private static int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    public MyLinkedList() {
+        size = 0;
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        Node<T> l = last;
+        Node<T> newNode = new Node<>(value, last, null);
+        last = newNode;
+        if (l == null) {
+            first = newNode;
+        } else {
+            l.next = newNode;
+        }
+        size++;
+        return true;
     }
 
     @Override
     public void add(T value, int index) {
-
+        if (index < 0 || index > size) {
+            throw new IndexOutOfBoundsException();
+        }
+        if (index == size) {
+            add(value);
+            return;
+        }
+        Node<T> lastNode = getNode(index);
+        Node<T> firstNode = last.prev;
```

Comment:
```suggestion
        Node<T> firstNode = lastNode.prev;
```
you probably wanted this?